tracing/src/main/java/com/palantir/remoting2/tracing/Tracer.java

```diff
@@ -51,14 +51,23 @@ protected Trace initialValue() {
     private static TraceSampler sampler = AlwaysSampler.INSTANCE;
 
     /**
+     * Creates a new trace, but does not set it as the current trace. The new trace is {@link
+     * Trace#isObservable observable} iff the given flag is true, or, iff {@code isObservable} is absent,
+     * if the {@link #setSampler configured sampler} returns true.
+     */
+    public static Trace createTrace(Optional<Boolean> isObservable, String traceId) {
```

Comment:
And maybe even remove the traceId parameter. Ideal might be we add a clearTrace method which returns a copy of the current trace and zeros the current state, then use that method in "withTrace" instead of copy/set. At that point, this method goes away.